project.clj

```diff
@@ -6,14 +9,32 @@
 
   :dependencies [[org.clojure/clojure "1.6.0"]
                  [org.clojure/tools.reader "0.8.9"]
+
                  [ring/ring-core "1.3.2"]
+                 [clj-time "0.7.0"]
                  [commons-io "2.4"]
+                 [commons-codec "1.9"]
+                 [slingshot "0.12.2"]
+                 [org.slf4j/slf4j-api "1.7.7"]
+
                  [bidi "1.18.9" :exclusions [org.clojure/clojurescript]]
                  [compojure "1.3.3"]
                  [prismatic/schema "0.4.0"]
-                 [puppetlabs/kitchensink "1.1.0"]]
+                 [puppetlabs/kitchensink ~ks-version]
+                 [puppetlabs/trapperkeeper-metrics "0.1.1-SNAPSHOT"]]
```

Comment:
Yeah, it's only being used on TK projects now.  If we avoid adding this dependency, we leave the option open for it to be used on non-TK projects.  We are probably not likely to have any of those in-house any time soon - I think that's the main reason for doing it this way.  My side of the coin is that it would be nice if non-PL consumers could use this without TK, but that's obviously less important than optimizing for internal consumption.